app/controllers/companies_controller.rb

```diff
@@ -0,0 +1,45 @@
+class CompaniesController < ApplicationController
+
+	before_action :authenticate_user!
+
+	def new
+
+	end
+
+	def create
```

Comment:
This method will find admin_id for the users, i.e. for particular user, which user is admin for that. I will remove some unnecessary code. Please review and suggest.